devices/HapticGlove/src/HapticGlove.cpp

```diff
@@ -537,22 +551,25 @@ class HapticGlove::SenseGloveImpl::SenseGloveHapticActuator : public wearable::a
 
     bool setHapticCommand(double& value) const override
     {
+        yError() << LogPrefix << "Wrong method has been called! To set the haptic command please use the setHapticsCommand method.";
+        return false;
+    }
```

Comment:
I would suggest to simply remove this method at this point

Reply:
Actually, I am wrong on this since it is an interface method. Just make sure that the message is not printed continuously in our use cases.

Reply:
> Actually, I am wrong on this since it is an interface method. Just make sure that the message is not printed continuously in our use cases.

Yes, actually I also tried to remove it back in time and eventually kept it for the same reason. Since it has not been called in glove devices the message is not being printed.